queries/ziggy/highlights.scm

```diff
@@ -0,0 +1,47 @@
+[
+  (true)
+  (false)
+] @constant.builtin.boolean
+
+(null) @constant.builtin
+
+[
+  (integer)
+  (float)
+] @constant.numeric
+
+
+(struct_field
+  key: (_) @keyword)
+  
+(struct
+  name: (_) @type)
+
+(tag) @function
+
+[
+  (string) 
+  (line_string)*
+] @string
+
+
+(comment) @comment.line
+
+(escape_sequence) @constant.character.escape
+
+(ERROR) @error
```

Comment:
```suggestion
```
We don't capture `(ERROR)` nodes; they're only useful for debugging and will lead to christmas trees when editing.